eoscostarica/guias.eoscostarica.io
# update(contributos): Add new contributors

### GH Issue
Resolves #174 
# 

### Steps to test
1. Open README
1. Check the contributors list

#### CheckList
